scripts/get_github_labels.py

```diff
@@ -4,14 +4,24 @@
 
 Parameters:
  --id=<id>: the number of the the GitHub Issue or Pull-Request
- --has-label=<label>: label to check (exit 0 if set, 1 unset), without
+ --has-label=<label>: label to check (exit 0 if set, 2 unset), without
                       --has-label, all labels of the Issue or Pull-Request
                       get printed.
+
+Exit codes:
+ 0: success
+ 1: any unexpected failure
```

Comment:
do we need to handle 1 in groovy script incase of any error?

Reply:
not really, check `!= 0` in the groovy script, which basically means: run e2e if not `0`